tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -137,46 +116,25 @@ def test_file_catalog_simulation(test_engine):
             ),
         ),
     ]
-    with Session(db.get_engine()) as session:
-        session.add_all(test_record)
-        session.commit()
-
-    return session
+    session.add_all(test_records)
+    session.commit()
 
 
-@pytest.fixture()
-def populate_status_tracking_table(test_engine):
+def _populate_status_tracking_table(session):
```

Comment:
should we rename this test function to match new table name?

Reply:
Yes, thanks! I also updated other old status tracking comments.